site/public/js/userform.js

```diff
@@ -300,6 +300,13 @@ function clearValidityWarnings() {
 }
 
 function closeButton() {
+  let form = document.querySelectorAll('form')[0];
+  // edit from manage page
+  if($('[name="edit_user"]', form).val() === "true"){
+    clearUserFormInformation();
+    $('#edit-user-form').css('display', 'none');
+    return;
```

Comment:
Adding this snippet here prevents the 'Changes to form will be lost' popup from showing up. To recreate, open up the edit user form and make any edit and hit cancel.

Reply:
Hii @shwehtom89 you are right,  no alert is shown for the popup opened from the inline pencil icon. In the PR #4996 changes were asked for the new grader/student popup only.  
However I think as per your suggestions we can expand alert message to this popup (pencil) also. 
Thanks.